contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+from future import standard_library # You would need to download future.
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
```

Comment:
@anirudhjain75 could you please help me understand the one at the end of the page?